src/workflows/pyfr/common/steps/Simulation/Start/index.js

```diff
@@ -43,6 +43,10 @@ const SimulationStart = React.createClass({
     };
   },
 
+  componentWillUpdate(nextProps, nextState) {
+    console.log(nextState.EC2);
+  },
+
```

Comment:
whoop, yes it should.

On Wed, May 4, 2016 at 10:31 AM, Chris Harris <notifications@github.com>
wrote:

> In src/workflows/pyfr/common/steps/Simulation/Start/index.js
> <https://github.com/Kitware/HPCCloud/pull/369#discussion_r62071095>:
>
> > @@ -43,6 +43,10 @@ const SimulationStart = React.createClass({
> >      };
> >    },
> >
> > +  componentWillUpdate(nextProps, nextState) {
> > +    console.log(nextState.EC2);
> > +  },
> > +
>
> Should this be removed?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Kitware/HPCCloud/pull/369/files/097e77fec9316fe07ade3c42a52952fbd508298f..0ba05b22600b862ea3c9a85a00981a9a6822b871#r62071095>
>



-- 
- Tristan Wright

*R&D Engineer*
*Kitware Inc.*